README.rst

```diff
@@ -93,5 +93,42 @@ Dynamic schedule:
         def get_next_schedule_item(cls, index):
             ...
 
+Alternatively, an entrypoint can be decorated with the `entrypoint_retry` decorator:
```

Comment:
Alternatively, an entrypoint can be decorated with the `entrypoint_retry` decorator,
to automatically retry the method if it raises certain exceptions: